default.yml

```diff
@@ -228,10 +228,14 @@ Lint/ConstantResolution:
 Rails/SaveBang:
   Enabled: false
 
-Rails/FindBy:
-  Enabled: true
-  Include:
-    - app/**/*.rb
+# Temporarily removing until Mongoid is no more...
+# Mongoid #find_by raises an exception if it can't be found, unlike AR #find_by, so .where().first is preferred
+# Rubocop can't tell the difference between the two
+#
+# Rails/FindBy:
```

Comment:
Instead of commenting this out, can we mark it disabled?

Reply:
Oh yeah, we need to actually

Reply:
Thank you